### PR TITLE
Added recipe for tangelo (mark 2)

### PR DESCRIPTION
**Pull request by pmlandwehr**
Fixed `ws4py` and `cherrypy` at appropriately earlier versions. Let's get this built and added...

### CODE REVIEW

File: recipes/tangelo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+
```
Comment (ocefpaf) [inline recipes/tangelo/meta.yaml]: It is a minor thing, but can you remove the extra line here?

**Comment by pmlandwehr** [comment]
(ping @ocefpaf)

**Comment by ocefpaf** [comment]
@pmlandwehr I am waiting `cliff` to be properly feedstocked before merging anything else. (Sorry, Travis-CI is flaky again :unamused:)

**Comment by pmlandwehr** [comment]
aaahhhh. Ok.

**Comment by jakirkham** [comment]
@ronichoudhury, just letting you know we have added `tangelo` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) (`conda install -c conda-forge tangelo`). The maintenance burden here is light as all testing releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this repo]( https://github.com/conda-forge/tangelo-feedstock ). If you have any interest in being added as a maintainer (to make new releases for instance), we would love to have you on board. Thanks for your work on `tangelo`. 😄

**Comment by ronichoudhury** [comment]
This is neat! Thanks for letting me know about it.

I am willing to be involved but I haven't used conda before (maybe once at a hackathon when Continuum IO folks were actively helping me use it) so just let me know what I can do to get on board.

**Pull request merged**